v-next/hardhat/src/internal/builtin-plugins/artifacts/artifacts-manager.ts

```diff
@@ -1,318 +1,405 @@
 import type {
   ArtifactsManager,
   GetAtifactByName,
-  BuildInfo,
 } from "../../../types/artifacts.js";
 
+import { EOL } from "node:os";
```

Comment:
Didn't know about this. We should probably start using it everywhere.

Reply:
TBH I only used it because it was there. I think node translates `\n` to EOL everywhere.